examples/8-eks-cluster-with-karpenter/provisioners/default_provisionar.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: karpenter.sh/v1alpha5
```

Comment:
Fix typo in file name `default_provisionar.yaml`, should be `default_provisioner.yaml`